From deff2b03aee9fe118dd7063427b5865de1ca0636 Mon Sep 17 00:00:00 2001
From: John Howard <howardjohn@google.com>
Date: Wed, 10 Nov 2021 16:27:11 -0800
Subject: tf: do not cleanup external ServiceEntry (#36009)

We do not clean up Service's, so cleaning up the ServiceEntry that is
for the same goals doesn't really make sense.

This only really impacts local debugging
---
 tests/integration/pilot/common/apps.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/integration/pilot/common/apps.go b/tests/integration/pilot/common/apps.go
index e02bafc4e2..940978372a 100644
--- a/tests/integration/pilot/common/apps.go
+++ b/tests/integration/pilot/common/apps.go
@@ -325,7 +325,7 @@ func SetupApps(t resource.Context, i istio.Instance, apps *EchoDeployments) erro
 	if err != nil {
 		return err
 	}
-	if err := t.ConfigIstio().ApplyYAML(apps.Namespace.Name(), se); err != nil {
+	if err := t.ConfigIstio().ApplyYAMLNoCleanup(apps.Namespace.Name(), se); err != nil {
 		return err
 	}
 	return nil
-- 
2.35.3

